### PR TITLE
Brushup Barbarian feats part 1

### PR DESCRIPTION
**Pull request by TikaelSol**
Levels 1-8 feats. I'll do the other half next.

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -1192,6 +1193,12 @@
                     "Drops": "Level 1 barbarian class feature",
                     "Prompt": "Select an instinct."
                 },
+                "InuredToAlchemy": {
+                    "CriticalSuccess": "Each critical success reduces the stage by 3 for a  alchemical poison, drug, or addiction.",
```
Comment (simonward) [inline static/lang/re-en.json:1197]: an alchemical

File: packs/data/feats.db/thrash.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
             "value": 1
         },
         "description": {
-            "value": "<p><strong>Requirements</strong> You have a foe @UUID[Compendium.pf2e.conditionitems.Grabbed]{Grabbed}</p>\n<hr />\n<p>You thrash the grabbed foe around. It takes bludgeoning damage equal to your Strength modifier plus your weapon specialization damage plus your Rage damage. The foe must attempt a basic Fortitude save against your class DC.</p>"
+            "value": "<p><strong>Requirements</strong> You have a foe @UUID[Compendium.pf2e.conditionitems.Grabbed]{Grabbed}</p>\n<hr />\n<p>You thrash the grabbed foe around. It takes bludgeoning damage equal to your Strength modifier plus your weapon specialization damage plus your Rage damage. The foe must attempt a @Check[type:fortitude|dc:resolve(@actor.classDCs.barbarian.dc.value)|bnasic:true] save against your class DC.</p>"
```
Comment (simonward) [inline packs/data/feats.db/thrash.json:13]: bnasic

File: packs/data/feat-effects.db/effect-ghost-wrangler.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "definition": [
+                    {
+                        "or": [
+                            "item:slug:fist",
+                            "item:slug:unarmed-attack"
```
Comment (simonward) [inline packs/data/feat-effects.db/effect-ghost-wrangler.json:25]: Do we want to check for unarmed category?

**Review by simonward — APPROVED** [review]

File: packs/data/feat-effects.db/effect-ghost-wrangler.json (hunk 3 above)
Comment (TikaelSol) [reply packs/data/feat-effects.db/effect-ghost-wrangler.json:25]: `Until the beginning of your next turn, your fists gain the effects of the ghost touch property rune` it wouldn't affect non-fist strikes. This is almost certainly being too selective but that's on Paizo for using that wording.

File: packs/data/feat-effects.db/effect-reckless-abandon-barbarian.json
Comment (simonward) [inline packs/data/feat-effects.db/effect-reckless-abandon-barbarian.json:1]: The AC penalty should probably be untyped rather than circumstance.

**Review by simonward — APPROVED** [review]

File: packs/data/feat-effects.db/effect-ghost-wrangler.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                        ]
+                    }
```
Comment (stwlam) [inline packs/data/feat-effects.db/effect-ghost-wrangler.json:27]:
```suggestion
                    "item:category:unarmed",
                    "item:base:fist"
```

Comment (TikaelSol) [reply packs/data/feat-effects.db/effect-ghost-wrangler.json:27]: One issue here is that this restricts it to just fist strikes, when most people would flavor their unarmed attack as a fist. The feat itself is truly awful regardless, but I doubt they are wanting to restrict it to only monk barbarians.

Comment (stwlam) [reply packs/data/feat-effects.db/effect-ghost-wrangler.json:27]: We could have the feat give a fist attack

Comment (TikaelSol) [reply packs/data/feat-effects.db/effect-ghost-wrangler.json:27]: I guess? It's one of the more useless feats so I kind of just want to stop having the PR held up by it.

**Pull request merged**